src/StyleSet.hpp

```diff
@@ -331,6 +331,7 @@ class StyleSet : public QObject
    * - __style__: `italic`, `upright`, `oblique`
    * - __weight__: `light`, `regular`, `demibold`, `bold`
    * - __capMode__: `mixedcase`, `alluppercase`, `alllowercase`, `smallcaps`, `capitalize`
+   * - __hinting__: `defaulthinting`, `nohinting`, `verticalhinting`, `fullhinting`
```

Comment:
It should be noted in the documentation above that this is an optional property coming after `weight` and before `size`